ui/src/Pages/VideoPlayer/Index.jsx

```diff
@@ -241,6 +240,40 @@ function VideoPlayer() {
     setPlayer,
   ]);
 
+  const play = useCallback(() => {
+    dispatch(
+      updateVideo({
+        idleCount: 0,
+      })
+    );
+
+    videoRef.current.play();
+  }, [dispatch, videoRef]);
+
+  const pause = useCallback(() => {
+    dispatch(
+      updateVideo({
+        idleCount: 0,
+      })
+    );
+    videoRef.current.pause();
+  }, [dispatch, videoRef]);
+
+  const playerPause = useCallback(
```

Comment:
the function name here makes no sense as this function doesnt just pause the video